packages/adminrouter/extra/src/test-harness/tests/test_metrics.py

```diff
@@ -35,6 +36,55 @@ def test_metrics_prometheus(self, master_ar_process):
         assert resp.headers['Content-Type'] == 'text/plain'
         assert resp.text.startswith('# HELP nginx_vts_info Nginx info')
 
+    def test_metrics_prometheus_static_upstreams_annotated(self, master_ar_process):
+        """
+        /nginx/metrics returns metrics in Prometheus format that are properly
+        annotated for static upstreams
+        """
+        # Adding upstreams present in both Open and EE DC/OS
+        static_upstream_annotations = {
```

Comment:
Can we use `pytest.mark.parametrize` here so each of these checks runs as a separate test?

Reply:
That also means we could have a separate test for the `repo_is_ee` case which lives in Enterprise - that seems better and more conventional!

Reply:
@jongiddy I have parametrized the test
@adamtheturtle with the test parametrized it would be exactly the same code/function for EE and Open with only difference being the parametrization list, so it seems to me as a code duplication. (that's why I haven't implemented it)

Reply:
I'd also like to see a test for Enterprise, even if it means to duplicate stuff initially. One day they will diverge (as almost anything in DC/OS EE).

Reply:
@timaa2k I have added Open- and EE-specific versions of the static endpoint annotation test, PTAL.